dispatch/apps/api/views.py

```diff
@@ -431,23 +431,32 @@ def get_queryset(self):
         return queryset
 
 
-@api_view(['POST'])
 @permission_classes((AllowAny,))
-def user_authenticate(request):
+class AuthenticationViewSet(viewsets.GenericViewSet):
```

Comment:
Do you think it might make sense to call this `TokenViewSet`? I'm undecided. 

We can treat a token like any other REST resource and then implement the `create` and `destroy` methods, which in this case would be synonymous with login and logout.

Let me know what you think